src/feedback/components/FeedbackForm.vue

```diff
@@ -86,18 +126,21 @@ export default {
       default: true,
     },
   },
+  data () {
+    return {
+      isDismissFeedbackDialogVisible: false,
+    }
+  },
   computed: {
     canSave () {
-      if (!this.isNew && !this.hasChanged) {
-        return false
-      }
-      return true
+      return this.isNew || this.hasChanged
```

Comment:
Haha, so much nicer than the previous convoluted logic! .. I guess the previous logic started off more complex, then got reduced into a really odd construct. :+1: